### ISSUE TITLE
scholarly.search_pubs does not return the full description of an abstract for a researched paper

### ISSUE THREAD

**Issue by danielkuang74** [scholarly-python-package/scholarly#543]
**Describe the bug**
I try to get the abstract out of one article using scholar.search_pubs function, but the response only returns partial of the abstract.

**To Reproduce**
from scholarly import scholarly

pub_title_list = ["Accelerated DNA methylation age and the use of antihypertensive medication among older adults", 
                  "A western diet pattern is associated with higher concentrations of blood and bone lead among middle-aged and elderly men"]

for pub_title in pub_title_list:
    pub_query = scholarly.search_pubs(pub_title)
    pub = next(pub_query)
    print(pub["bib"]["abstract"])
    print("\n") 

**Expected behavior**
get the full abstract description of these two articles.
the abstract of the first article is "The discrepancy of DNA methylation age (DNAmAge) with chronological age (termed as age acceleration, AA) has been identified to be associated with many aging-related health outcomes including hypertension. Since taking antihypertensive medication (AHM) could prevent aging-related diseases caused by hypertension, we hypothesized that using AHM could also reduce the AA. We examined this hypothesis among 546 males aged 55-85 years by exploring the associations of AHM use with AA and its change rate (ΔAA) in two visits with a median follow-up of 3.86 years. Horvath DNAmAge was derived from DNA methylation profiles measured by Illumina HumanMethylation450 BeadChip and information on AHM use was collected by physician interview. A general decreasing pattern of AA was observed between the two visits. After the fully adjusting for potential covariates including hypertension, any AHM use showed a cross-sectional significant association with higher AA at each visit, as well as a longitudinal association with increased ΔAA between visits. Particularly, relative to participants who never took any AHM, individuals with continuous AHM use had a higher ΔAA of 0.6 year/chronological year. This finding underlines that DNAmAge and AA may not be able to capture the preventive effects of AHMs that reduce cardiovascular risks and mortality."

the abstract of the second article is "Background: Little is known about the effects of overall dietary pattern on lead concentration.Objective: We examined the association of overall dietary patterns, derived from a semiquantitative food frequency questionnaire, with bone and blood lead concentrations.Methods: These longitudinal analyses included mostly non-Hispanic white, middle-aged-to-elderly men from the Veterans Affairs Normative Aging Study. Long-term lead exposures were measured as tibia and patella lead concentrations by using K-shell-X-ray fluorescence. Short-term lead exposures were measured as blood lead concentrations by using graphite furnace atomic absorption spectroscopy. Dietary pattern scores were derived by using factor analysis. Linear mixed-effects models were utilized to predict blood lead concentrations among 983 men, aged 44-92 y at baseline, with a total of 3273 observations (during 1987-2008). We constructed linear regression models to determine the relations between dietary patterns and bone lead concentrations among 649 participants with an age range of 49-93 y.Results: Two major dietary patterns were identified: a prudent dietary pattern, characterized by high intakes of fruit, legumes, vegetables, whole grains, poultry, and seafood; and a Western dietary pattern, characterized by high intakes of processed meat, red meat, refined grains, high-fat dairy products, French fries, butter, and eggs. After adjusting for age, smoking status, body mass index, total energy intake, education, occupation, neighborhood-based education and income level, men in the highest tertile of the Western pattern score (compared with the lowest) had 0.91 μg/dL (95% CI: 0.41, 1.42 μg/dL) higher blood lead, 5.96 μg/g (95% CI: 1.76, 10.16 μg/g) higher patella lead, and 3.83 μg/g (95% CI: 0.97, 6.70 μg/g) higher tibia lead. No significant association was detected with the prudent dietary pattern in the adjusted model.Conclusions: These findings suggest that the Western diet is associated with a greater lead body burden among the middle-aged-to-elderly men. More studies are needed to examine the underlying mechanisms by which dietary patterns are associated with lead concentrations."

**Screenshots**
the first article only returns "The discrepancy of DNA methylation age (DNAmAge) with chronological age (termed as age acceleration, AA) has been identified to be associated with many aging-related health outcomes including hypertension. Since taking antihypertensive medication (AHM) could prevent aging-related diseases caused by hypertension, we hypothesized that using AHM could also reduce the AA. We examined this hypothesis among 546 males aged 55–85 years by exploring the associations of AHM use with AA and its change rate (Δ AA) in two"

the second article only returns "Background: Little is known about the effects of overall dietary pattern on lead concentration. Objective: We examined the association of overall dietary patterns, derived from a semiquantitative food frequency questionnaire, with bone and blood lead concentrations. Methods: These longitudinal analyses included mostly non-Hispanic white, middle-aged-to-elderly men from the Veterans Affairs Normative Aging Study. Long-term lead exposures were measured as tibia and patella lead concentrations by using K-shell-X-ray fluorescence"

**Desktop (please complete the following information):**
 - python version: 3.9.13
 - OS: Window 11

**Do you plan on contributing?**
Your response below will clarify whether the maintainers can expect you to fix the bug you reported.
- [ ] No since I do not know how to fix it

**Additional context**
Any other scholarly function can satisfy my request.

**Comment by neptunus123** [comment]
I agree. The search_pubs method does not return any response. It is really important to use it for references without DOI links. So please fix it.

**Comment by Xwindwhile** [comment]
The correct class name that contains full of abstract should be 'gsh_csp' (`<div class="gsh_csp">`), while it is 'gs_rs' in Scholarly (class PublicationParser; def _scholar_pub).
